src/main/java/io/cdap/plugin/http/source/batch/HttpInputFormatProvider.java

```diff
@@ -17,67 +17,24 @@
 package io.cdap.plugin.http.source.batch;
 
 import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import io.cdap.cdap.api.data.batch.InputFormatProvider;
-import io.cdap.plugin.http.source.common.BaseHttpSourceConfig;
 
 import java.util.Map;
 
 /**
  * InputFormatProvider used by cdap to provide configurations to mapreduce job
  */
 public class HttpInputFormatProvider implements InputFormatProvider {
+  public static final String PROPERTY_CONFIG_JSON = "mapreduce.config.json";
```

Comment:
By convention, the mapreduce prefix is for properties from the mapreduce framework/libraries. I think it's better to use something like 'cdap.http.config'

Reply:
done